Лаб 1/1.py

```diff
@@ -0,0 +1,19 @@
+numbers = [2, -93, -2, 8, None, -44, -1, -85, -14, 90, -22, -90, -100, -8, 38, -92, -45, 67, 53, 25]
+
+# TODO заменить значение пропущенного элемента средним арифметическим
+
```

Comment:
В рамках данной задачи мы ищем значение `None` руками, что до изучения методов является корректным, но далее можно будет воспользоваться методом поиска индекса искомых элементов списка: 
```suggestion
none_index = numbers.index(None)
```
Тогда мы всегда будем находить индекс элемента `None`, даже если он будет перемещаться по списку